docs/content/en/docs/tasks/troubleshoot/_troubleshooting.md

```diff
@@ -190,6 +190,10 @@ When you get the external IP you can ssh into the nodes using the private ssh ke
 ssh -i <ssh-private-key> <ssh-username>@<external-IP>
 ```
 
+### "msg"="discovered IP address" log message with an address value of the controlplane in the capv log
```

Comment:
```suggestion
### "msg"="discovered IP address" log message with an address value of the controlplane
```

Or specify which pod log you found this message.

Reply:
I think we should add this under the [timed-out-waiting-for-the-condition-on-clusters](https://anywhere.eks.amazonaws.com/docs/tasks/troubleshoot/_troubleshooting/#timed-out-waiting-for-the-condition-on-clustersyour-cluster-name) section, since that's the error users will see cli exit with. And in that section add this as one of the things to check for:
Log message `"msg"="discovered IP address"` with an address value of the controlplane in either:
1.  the file `${CLUSTER_NAME}/logs/capv-controller-manager.log` 
2. Using the kind cluster's kubeconfig to check the capv pod logs:
```
export KUBECONFIG=${PWD}/${CLUSTER_NAME}/generated/${CLUSTER_NAME}.kind.kubeconfig
kubectl logs -f -n capv-system -l control-plane="controller-manager" -c manager
```